src/pages/RepoPage/CoverageTab/Navigator/Navigator.tsx

```diff
@@ -0,0 +1,67 @@
+import { useHistory, useLocation } from 'react-router-dom'
+
+import { useNavLinks } from 'services/navigation'
+import { RadioTileGroup } from 'ui/RadioTileGroup'
+
+const TABS = {
+  overview: 'Overview',
+  flags: 'Flags',
+  components: 'Components',
+} as const
+type TabsValue = (typeof TABS)[keyof typeof TABS]
+
+function getInitialSelection(path: string) {
+  if (path.includes('/flags')) {
```

Comment:
For added clarity and to prevent any unexpected behavior with paths, I would suggest checking if the current location precisely matches the path. For example, `location.pathname === flags.path()` retrieved from useNavLinks